ironfish-cli/src/multisigBroker/server.ts

```diff
@@ -307,6 +307,34 @@ export class MultisigServer {
       return
     }
 
+    // If the session is in the collecting identities phase, we can safely
+    // remove a disconnected client's identity from the list. Otherwise, if a
+    // client disconnects and then reconnects, they can possibly be counted as
+    // multiple identities, leaving the session in a bad state.
+    if (client.identity != null) {
+      if (isDkgSession(session)) {
+        if (
+          session.status.round1PublicPackages.length === 0 &&
+          session.status.round2PublicPackages.length === 0
```

Comment:
nit - we could probably get away with just checking `round1PublicPackages` here

this does make me wonder if we want the session to act more like a state machine and track which round/step it's on. we might be able to use that to force retries of round1, too if we blocked round1 package submissions if the state doesn't match